redgifs/enums.py

```diff
@@ -60,6 +60,17 @@ def random(cls, count: int = 1) -> Optional[List["Tags"]]:
             raise ValueError('n cannot be more than 4664')
         return random.choices([x for x in Tags.__members__.values()], k=count)
 
+    @classmethod
+    def single_random(cls) -> "Tags":
+        """
+        A class method to get a random tag.
```

Comment:
```suggestion
        A class method to get a single random tag.
```
Might be good to just specify it in the docs